### PR TITLE
LOK-2429: Add option to enable and disable debug port

### PR DESCRIPTION
**Pull request by Umairk95**

### CODE REVIEW

File: lokahi-minion/templates/minion-deployment.yaml [hunk 1]
```diff
@@ -89,7 +89,11 @@ spec:
               value: "/opt/karaf/certs/CA.cert"
             {{- end}}
             - name: JAVA_TOOL_OPTIONS
-              value: "-Djava.net.preferIPv4Stack=true -XX:MaxRAMPercentage=50 -javaagent:agent/opentelemetry-javaagent.jar" #-agentlib:jdwp=transport=dt_socket,server=y,address=5005,suspend=n,quiet=y"
```
Comment (mershad-manesh) [inline lokahi-minion/templates/minion-deployment.yaml:92]: Is there a reason why we have removed `-Djava.net.preferIPv4Stack=true` ?

Comment (Umairk95) [reply lokahi-minion/templates/minion-deployment.yaml:92]: yes i missed this one, now added.

**Comment by mershad-manesh** [comment]
Make sure to  add the new property to the README files

**Comment by mershad-manesh** [comment]
Does it make sense to allow the user to change the debug port (and/or other properties)?

**Comment by Umairk95** [comment]
> Does it make sense to allow the user to change the debug port (and/or other properties)?

Yes we can allow user to change the debug port.

File: lokahi-minion/values.yaml [hunk 2]
```diff
@@ -19,6 +19,8 @@ imageShortName: lokahi-minion
 imagePullPolicy: IfNotPresent
 minionAppSecrets: ""
 replicas: 1
+debug: false
```
[the remark this answers is not in the document]
Comment (Umairk95) [reply lokahi-minion/values.yaml]: yes, you are right .
I have added properties, i think arguments option is not necessary

**Review by mershad-manesh — APPROVED** [review]
Took a quick look

**Pull request merged**